lib/riak/crdt/counter.ex

```diff
@@ -12,7 +12,8 @@ defmodule Riak.CRDT.Counter do
   @doc """
   Increment a `counter` on the `amount` defaulting in 1
   """
-  def increment(counter, amount \\ 1) when Record.is_record(counter, :counter) do
+  def increment(counter, amount \\ 1)
```

Comment:
It prevents from having a warning at compile time on that function because it has default value for the 'amount' param.
I believe this warning is issued since elixir 1.2